sources/stylesheets/_general.scss

```diff
@@ -47,11 +50,33 @@ main {
     rotate: 45deg;
 
     path {
-      stroke: white;
+      stroke: $white;
     }
   }
 
+  &:after {
+    content: attr(data-title);
+    opacity: 0;
+    transition: opacity .3s ease-in-out;
+    padding: 8px;
+    width: fit-content;
```

Comment:
In Firefox its wrapped:

![Screenshot 2023-12-11 at 18 27 52](https://github.com/Voog/design-faro/assets/378755/e968ad48-7433-4055-8550-664fd8450f50)
![Screenshot 2023-12-11 at 18 27 45](https://github.com/Voog/design-faro/assets/378755/e68e2819-11ba-49b2-9a43-c84e767b46eb)